src/sparql/orcid-contributor-violation.sparql

```diff
@@ -0,0 +1,10 @@
+PREFIX dcterms: <http://purl.org/dc/terms/>
+
+SELECT ?subject ?orcid
+WHERE {
+  ?subject dcterms:contributor ?orcid .
+  FILTER(str(?orcid) != "https://github.com/orgs/pato-ontology/teams/pato-community")
```

Comment:
Allow this in general, st like: `(regex(?orcid,"https://github.com/orgs/[a-z-_]/teams/"))`

I like the ability to credit teams.

Reply:
Awesome!